modules/va_facilities/README.v1.yml

```diff
@@ -0,0 +1,599 @@
+openapi: '3.0.0'
+info:
+  version: 1.0.0
+  title: VA Facilities
+  description: |
+
+    ## Background
+
+    This API provides information about physical VA facilities. Information available includes
+    geographic location, address, phone, hours of operation, and available services. 
+
+    VA operates several different types of facilities, the types represented in this API include:
```

Comment:
Small grammar nitpick (comma splice). Should be "VA operates several different types of facilities. The types represented in this API include:"

Reply:
:+1: I'm not changing v0. That way folks can see what an impact we've made in our time here. :smiley: